lisa/main.py

```diff
@@ -58,6 +59,11 @@ def main() -> int:
 
         log.info(f"Python version: {sys.version}")
         log.info(f"local time: {datetime.now().astimezone()}")
+
+        # We don't want command line args logging to leak any provided
+        # secrets, if any ("s:key:value" syntax)
+        add_secrets_from_pairs(args.variables)
```

Comment:
It looks call `_load_from_pairs` can get same result. How about make it public, and call it here?

Reply:
It will leak a dict, unused. Kind ugly, no?

Reply:
It's not a leak, the returned dict reference is not hold somewhere globally. It's like `raw_pair.lower()`. I asked it, because it can share more logic. Anyway, it doesn't copy many logic. Feel free to choose a way.

Reply:
The original name will be misleading, if we just remove _. I'll think about it soon, dealing with other stuff in the meantime.